### ISSUE TITLE
Issues by clicking in "terms and conditions" and "privacy policy" link

### ISSUE THREAD

**Issue by OrtizBetD** [gitbito/bitoai#82]
I have downloaded the plugin in IntelliJ on 05.06.2023. Before creating a workspace, I click on the "terms and conditions"  and "privacy policy" I get just a blank window, please check

**Comment by KumarVariable** [comment]
Hello @OrtizBetD ,

Thanks for reporting the issue.

Can you please share the details below:
1) BITO version
2) IntelliJ IDEA version
3) Operating System

Alternatively, you can also check 
1) On clicking on the 'Terms and Conditions link from Create a Workspace, the pop-up window is not getting blocked
2) The URL 'https://bito.ai/termsofservice/' and 'https://bito.ai/privacy-policy/' is not blocked by any firewall services

Please reach out for any further assistance

**Comment by OrtizBetD** [comment]
Hello @KumarVariable  thanks for your message. When I click in the urls you provided I can  access to them without issue. The problem occurs when I click on the links from intellij. here you find the details:
1. BITO Version: 1.0.138
2. IntelliJ Version: 2022.1.4
3. Operating System: Windows 11 Home.

Thanks so much, looking forward to hear from you

**Comment by nisha-bito** [comment]
hi @OrtizBetD ,

Can you please download latest version of Bito 1.1.0. We have changed the create workspace flow a bit and you should be able to create workspace now.
Please suggest of you come across any issue.

thanks,
Nisha

**Comment by nisha-bito** [comment]
hi @OrtizBetD ,

Any update the issues, do you still see an issue with new Bito plugin 1.1.2? 
Please note, this ticket will be auto closed in few days, please feel free to create a new ticket in case the problem still exists. 

Thanks,
Nisha
